src/types/onyx/Fund.ts

```diff
@@ -1,3 +1,4 @@
+import {BankName} from '@components/Icon/BankIcons';
```

Comment:
Can we move BankName to separate file (or maybe CONST)? A bit weird to import name from icon

Reply:
I'll move BANK_NAMES to the const, but I need to figure out where to put types.

Reply:
I think `src/types/onyx` would be a good place, `AccountData` for example? 😄